python/chronos/example/serving/readme.md

```diff
@@ -0,0 +1,54 @@
+# Serving Forecaster Example
+
+Model serving is most flexible in terms of model deployment and it can generate predictions on data in real-time and online.
```

Comment:
Model serving is for model deployment in a production environment for wide accessibility.